inlineplz/interfaces/github.py

```diff
@@ -41,15 +41,15 @@ def __init__(self, owner, repo, pr=None, branch=None, token=None, url=None, comm
         self.owner = owner
         self.repo = repo
 
-        github_repo = self.github.repository(self.owner, self.repo)
-        all_commits = self.repo_commits(github_repo)
+        self.github_repo = self.github.repository(self.owner, self.repo)
+        all_commits = self.repo_commits(self.github_repo)
         self.master_sha = all_commits[0].sha
         print('Master SHA: {0}'.format(self.master_sha))
 
         print('Branch: {0}'.format(branch))
+        self.pr = None
```

Comment:
`prospector: pylint: Attribute name "pr" doesn't conform to snake_case naming style (invalid-name)`